pos_session_pay_invoice/models/pos_order.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Jose Zambudio - Aures Tic <jose@aurestic.es>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+from odoo.osv.expression import AND
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    @api.model
+    def search_paid_order_ids(self, config_id, domain, limit, offset):
```

Comment:
Hi. Why this change in that module. I'm not sure to understand why this overload is interesting, but my question is why in this module ?

Reply:
This code is present in V14. Please refer to this PR for the changes that were added: https://github.com/OCA/pos/pull/1006.
Let me know if this code is useful or if it should be removed.

Reply:
Reading the PR, the error is raised with orders created in this module, but now orders aren’t created; only `account.bank.statement.line` are created. So, I think the code is unused. Let me know what you think, please.

Reply:
indeed, I think this part of code is now obsolete.

Reply:
Done, I removed the code.

Reply:
thanks !